src/discussions/posts/post/Post.jsx

```diff
@@ -0,0 +1,111 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { faQuestionCircle, faStar as faEmptyStar } from '@fortawesome/free-regular-svg-icons';
+import {
+  faCircle, faComments, faFlag, faStar as faSolidStar, faThumbtack,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Image } from '@edx/paragon';
+import { Routes } from '../../../data/constants';
+import messages from './messages';
+
+function Post({ post, intl }) {
+  return (
+    <Link
+      className="discussion-post d-flex list-group-item p-2 text-decoration-none text-gray-900"
+      data-post-id={post.id}
+      to={
+        Routes.COMMENTS.PATH.replace(':courseId', post.course_id)
+          .replace(':topicId', post.topic_id)
+          .replace(':postId', post.id)
+      }
+    >
+      <div className="d-flex post-unread-status m-1">
+        { post.read || <FontAwesomeIcon icon={faCircle} className="text-success-300" /> }
+      </div>
+      <div className="d-flex flex-column">
+        <div className="d-flex post-header">
+          <div className="d-flex user-avatar col-2 m-1 px-0">
+            <Image
+              className="w-75"
+              src="https://source.unsplash.com/100x100/?nature,flower"
+              roundedCircle
+            />
+          </div>
```

Comment:
```suggestion
          <Image
            src="https://source.unsplash.com/100x100/?nature,flower"
            roundedCircle
            className="col-1 my-auto p-0"
          />
```

Reply:
The image should not be hardcoded, it should be coming from the parent component. It can be hardcoded at some point, but not here.